README.md

```diff
@@ -43,23 +36,16 @@ To get a local copy up and running follow these simple example steps.
 
```

Comment:
- [ ] Add the necessary setup instructions like cloning and opening the index page